sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/PlanParserSuite.scala

```diff
@@ -1412,6 +1412,31 @@ class PlanParserSuite extends AnalysisTest {
     assertEqual("select a, b from db.c; ;;  ;", table("db", "c").select($"a", $"b"))
   }
 
+  test("table valued function with named arguments") {
+    // All named arguments
```

Comment:
Can you add some negative test cases that return errors? You can look here [1] for some examples, and here are some I can think of:

```
select * from my_tvf(arg1 -> 'value1')
select * from my_tvf(arg1 = => 'value1')
select * from my_tvf(arg1 ==> 'value1')
select * from my_tvf((arg1 => 'value1'))
select * from my_tvf(arg1 'value1')
select * from my_tvf(arg1 => )
select * from my_tvf(arg1 => , 42)
select * from my_tvf(my_tvf.arg1 => 'value1')
select * from my_tvf(arg1 => table t1)
select * from my_tvf(group => 'abc')
```

The last one checks that a reserved keyword like `GROUP` or `ORDER` will work as a named argument name.

[1] https://github.com/google/zetasql/blob/master/zetasql/parser/testdata/named_arguments.test

Reply:
Great, thanks!